lib/matplotlib/tests/test_backend_svg.py

```diff
@@ -107,6 +108,43 @@ def test_bold_font_output():
     ax.set_title('bold-title', fontweight='bold')
 
 
+# except for the first two lines, this is copy&paste from "def test_visibility"
+def _test_determinism(i):
+    from matplotlib import rc
+    rc('svg', hashsalt='asdf')
+
+    fig = plt.figure()
+    ax = fig.add_subplot(111)
+
+    x = np.linspace(0, 4 * np.pi, 50)
+    y = np.sin(x)
+    yerr = np.ones_like(y)
+
+    a, b, c = ax.errorbar(x, y, yerr=yerr, fmt='ko')
+    for artist in b:
+        artist.set_visible(False)
+
+    with BytesIO() as fd:
+        fig.savefig(fd, format='svg')
+
+        fd.seek(0)
+        buf = fd.read()
+        return buf
+
+
+@cleanup
+def test_determinism():
+    # The function "map_random_hash_seeds" requires
+    # "multiprocessing.get_context", which is available since Python 3.4.
+    # If the Python version is below 3.4, do not run the test.
+    import sys
+    if sys.version_info[:2] < (3, 4):
+        return
```

Comment:
You should probably Raise a skip test here rather than just returning true

See https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/tests/test_rcparams.py#L111